locale/es_ES/user.xml

```diff
@@ -35,7 +35,7 @@
 	<message key="user.feminine">F</message>
 	<message key="user.firstName">Nombre</message>
 	<message key="user.gender">Sexo</message>
-	<message key="user.initialsExample">Luz Ruiz Plasencia = LRP</message>
+	<message key="user.initialsExample">Juan Anselmo Seguí = JAS</message>
```

Comment:
Just curious, why does this PR include a name change?

Reply:
I don't know, maybe @marcbria can explain that. The only difference I can see is Anselmo is a second name and Ruiz is a last name, in this commit the most important chains are for API key feature

Reply:
@asmecher you catch us. :-)

During the translation sometimes you get bored. Lots of files, sometimes mechanic tasks... you know.

When we took this file to add the missing API chains, we noticed the sample name was changed in a former commit, so thought was funny changing this name in every release.

It's not an important change. As "Luz Ruiz Placencia" our new proposal is also a non existing name as is "Alan Smithee" but in latin context. 

In short, if you prefer keeping changes to minimal, "Luz Ruiz..." is a good name to keep.

Reply:
Take a look to the comments around this file:
"Secondary minor change to keep initials, but modify default name to a latin, with a non existing name. (Would be nice a tribute to "Aaron Hillel Swartz" here for all langs?)."

But thinking it twice, including Aaron's name in every OJS will hide his fingerprint in google... and hide Aaron's publications it's not the best way to tribute. :-)

Reply:
Very happy to let the occasional wink sneak into the software -- for example, I always make sure to use `abbrev` instead of `abbreviation` in the code. Just making sure someone hadn't snuck their own name into it for fame and fortune.